packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -136,7 +144,7 @@ func (aea *AeaApi) Init() error {
 	if uri_public != "" {
 		parts = strings.SplitN(uri_public, ":", -1)
 		if len(parts) < 2 {
-			fmt.Println("[aea-api  ][error] malformed Uri:", uri)
+			fmt.Println("[aea-api  ][error] malformed Uri:", uri_public)
```

Comment:
Was this change intentional? If so, comment should be `malformed Public URI:`

Reply:
it'll be redundant, as the uri will be printed.
there is a test in python class to catch user-input malformed uris